src/helper-modules/bpmn-helper/src/getters.js

```diff
@@ -1133,6 +1133,44 @@ function convertISODurationToMiliseconds(isoDuration) {
   return durationInMiliseconds;
 }
 
+/**
+ * An object containing all necessary values for duration
+ *
+ * @typedef DurationValues
+ * @type {object}
+ * @property {number | null} years
+ * @property {number | null} months
+ * @property {number | null} days
+ * @property {number | null} hours
+ * @property {number | null} minutes
+ * @property {number | null} seconds
+ */
+
+/**
+ *
+ * @param {DurationValues} durationValues with number of years, months, days, hours, minutes and seconds
+ * @returns {string} ISO duration string
+ */
+function calculateTimeFormalExpression(durationValues) {
```

Comment:
The functions in this file are meant to be used to get information from bpmn files. So it might make more sense to define this function somewhere else